tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -320,3 +318,138 @@ def test_event_bus_to_event_bus_cross_account_region(
         PolicyName=policy_name_bus_primary_to_bus_secondary,
     )
     aws_client.iam.delete_role(RoleName=role_name_bus_primary_to_bus_secondary)
+
+
+class TestEventsCrossAccountRegion:
+    @markers.aws.validated
+    @pytest.mark.skipif(is_old_provider(), reason="Not supported in v1 provider")
+    @pytest.mark.parametrize("cross_scenario", ["account"])
```

Comment:
I guess the plan is to add other scenarios here as well, right?
A short list in the PR description could clarify the scope of this vs. the follow-up PR (we discussed them via PM)